README.rst

```diff
@@ -216,55 +216,52 @@ In a nutshell:
 .. _cartridge-cli-creating_an_application_from_template:
 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Creating an application from template
+Creating an application from a template
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To create an application from the Cartridge template, say this in any directory:
+To create an application from a Cartridge template, run ``cartridge create`` in any directory:
 
-.. code-block:: bash
+..  code-block:: bash
 
     cartridge create [PATH] [flags]
 
-The following options (``[flags]``) are supported:
-
-.. // Please, update the doc in cli/commands on updating this section
-
-* ``--name string`` is an application name.
+Supported options (``[flags]``):
 
-* ``--from DIR`` is a path to the application template (see details below).
+.. // Please update the doc in cli/commands on updating this section
 
-* ``--template string`` is a name of application template to be used.
-  Currently only ``cartridge`` template is supported.
+* ``--name string`` – application name.
```

Comment:
Dashes are hard to discern when one's looking at a diff in monospace font. Instead of explicit m- and n-dash characters we can use triple and double hyphens:

```suggestion
*    ``--name string`` --- application name.
```

Reply:
Aren't we using n-dashes in English texts? If we are, then we have to use a double dash `--`, correct?